src/layout/strategy/container.js

```diff
@@ -3,10 +3,12 @@ export default function() {
         containerHeight = 0;
 
     var container = function(point) {
-        return !(point.x < 0 || point.y < 0 ||
+        // If the bounds haven't been defined, then don't enforce
```

Comment:
I'm not keen on the conflation of `0` and `falsey`. If `0` is a valid value then we should check against `null` otherwise we should explicitly check against `0`.